test/unit/pdf/finance/invoice_generator_test.rb

```diff
@@ -27,18 +25,27 @@ class Pdf::Finance::InvoiceGeneratorTest < ActiveSupport::TestCase
     assert_equal 'application/pdf', attachment.content_type
   end
 
-  test 'should generate valid PDF content with logo and line items' do
+  test 'should generate valid PDF content with jpg logo and line items' do
+    test_generate_with_logo_and_lines(file_fixture("wide.jpg").to_s)
+  end
+
+  test 'should generate valid PDF content with png logo and line items' do
+    test_generate_with_logo_and_lines(file_fixture("wide.png").to_s)
+  end
+
+  private
+  def test_generate_with_logo_and_lines(image_path)
```

Comment:
Another one related with style guides: We're missing a blank line after `private`.

Reply:
I think that the only way to enforce such requirement is to configure code-climate to complain about it. We don't have this consistent through codebase. It also makes sense to me to require an empty line. But I have some recollection that actually codeclimate complains about having it.

btw same with long lines. If we want this style, we need to setup codeclimate. I'm fine with long lines though in general.

```
  coffeelint:
    enabled: true
    checks:
      max_line_length:
        enabled: false
```

rubocop:
```
Layout/LineLength:
  Max: 99
  Enabled: false

Layout/EmptyLinesAroundAccessModifier:
  Enabled: false
```

I don't know historic reasons for these. As I said we can always discuss.

Reply:
There's a plugin https://docs.codeclimate.com/docs/rubocop for CodeClimate so that it reads from our `.rubocop` file.
Also, there are other ways of doing it, like having overcommit setup... Let's have a discussion round with the team about this as well :v: